src/components/JobsPage/Jobs.js

```diff
@@ -116,21 +116,29 @@ const Jobs = ({
   }
 
   const handleRerunJob = async job => {
-    const functionParts = job.function.split('/')
+    const [project, func] = job.function ? job.function.split('/') : ['', '']
```

Comment:
This is okay, but I think there is a way to make it better.
If `job.function` equals `'abcd'` (or any other non-empty string that does not include the character `/`) the `split('/')` will return `['abcd']`.
This will result in `project` equals `'abcd'` but `func` equals `undefined`. This is not what you wanted here. You wanted `func` to default to `''` in this case.

My suggestion is:
```suggestion
    const [project = '', func = ''] = job?.function?.split('/') ?? []
```
Giving the default values as part of the array destructuring.
This way if `job.function` equals `'abcd'`, `project` equals `'abcd'` and `func` equals `''`, as expected.

Note that the `?? []` I added at the end is important, because array destructuring throws if the right-hand side of the `=` is not an array. For example:

```js
const [a, b] = 150
VM240:1 Uncaught TypeError: 150 is not iterable
    at <anonymous>:1:16
```

If we omit the `?? []` and `job` or `job.function` is nullish (meaning `null` or `undefined`),  the expression would give:
```js
const [project = '', func = ''] = undefined
```
which will throw.
On the other hand, when the `?? []` is in place, and `job` or `job.function` is nullish, we will get:
```js
const [project = '', func = ''] = []
```
which will properly set `project` to `''` and `func` to `''`.